src/main/java/com/bench/stockmanagement/StockmanagementApplication.java

```diff
@@ -11,9 +14,26 @@ public class StockmanagementApplication {
 
     public static void main(String[] args) {
         SpringApplication.run(StockmanagementApplication.class, args);
-        System.out.println("Started...");
 
-        DynamoDbManager.createTable("Orders", 10L, 5L, "id", "S", "date", "S");
-        DynamoDbManager.createTable("SoldItems", 10L, 5L, "id", "S", "receiptNumber", "N");
+    }
+
+    private static void loadSelling() {
```

Comment:
where are these private methods invoked from?

Reply:
I used them here in the main method, will remove them when find out the final structure